components/raftstore/src/store/peer.rs

```diff
@@ -1072,46 +1072,19 @@ where
     {
         for msg in msgs {
             let msg_type = msg.get_msg_type();
-            let snapshot_index = msg.get_request_snapshot();
-            let i = self.send_raft_message(msg, trans) as usize;
-            match msg_type {
-                MessageType::MsgAppend => metrics.append[i] += 1,
-                MessageType::MsgAppendResponse => {
-                    if snapshot_index != raft::INVALID_INDEX {
-                        metrics.request_snapshot[i] += 1;
```

Comment:
Why this is removed?

Reply:
Because this msg is never used. If I keep it, the `add` function of `RaftSendMessageMetrics` must add one more argument. I think it's better to use a new msg type for this msg in the future instead of a special judge.